src/components/alphabet/images-space/ImagesSpace.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <div class="images-space">
```

Comment:
For css names i would use http://getbem.com/introduction/ Its kinda hard to learn and use but its great for readability